### PR TITLE
Users/aedler/cash register integration patches

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md [hunk 1]
```diff
@@ -92,15 +92,15 @@ Once the queue has been closed with a stop receipt, no hashing and signing of re
 
 #### End of Failure Receipt (Collective Failure Report)
 
-If for technical reasons signatures cannot be created by the fiskaltrust.SecurityMechanism, receipts need to be issued (according to the RKSV) and marked with a comment "security mechanism failed". Once the technical failure has been resolved, a signed collective receipt must be issued to make up for the signature linking of all receipts issued during the technical failure.
+If for technical reasons signatures cannot be created by fiskaltrust.SecurityMechanisms, receipts need to be issued (according to the RKSV) and marked with a comment "security mechanism failed". Once the technical failure has been resolved, a signed collective receipt must be issued to make up for the signature linking of all receipts issued during the technical failure.
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md:95]: If, for technical reasons, signatures cannot be..

File: doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md [hunk 2]
```diff
@@ -92,15 +92,15 @@ Once the queue has been closed with a stop receipt, no hashing and signing of re
...
 
-Further, you can find two fundamentally different types of failure distinguished by the fiskaltrust.SecurityMechanism:
+Further, you can find two fundamentally different types of failure distinguished by fiskaltrust.SecurityMechanisms:
```
Comment (DoxaPap) [inline doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md:97]: Furthermore/further down

File: doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md (hunk 1 above)
Comment (aedler-ft) [reply doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md:95]: done

File: doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md (hunk 2 above)
Comment (aedler-ft) [reply doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md]: done!

**Pull request merged**